script/foreman-start-dev

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-./node_modules/.bin/webpack-dev-server-without-h2 --config config/webpack.config.js --host "::" $WEBPACK_OPTS &
+npx webpack --config config/webpack.config.js --watch $WEBPACK_OPTS &
 ./bin/rails server -b \[::\] "$@"
```

Comment:
a tad unrelated, but isn't this script doing essentially the same as `bundle exec foreman start`, and could be just dropped?